study-builder/studies/study-example.conf

```diff
@@ -893,5 +893,18 @@
       "dispatchToHousekeeping": "false",  # Action handled synchronously.
       # ...
     },
+
+    # Update DSM workflow status.
+    #
+    # This action can define custom workflow status which will be exported to DSM
+    {
+      "action": {
+        "type": "UPDATE_CUSTOM_WORKFLOW",
```

Comment:
Looks like things are sorted here. Can you move this block up?

Reply:
Fixed